src/main/java/org/ladocuploader/app/inputs/LaDigitalAssister.java

```diff
@@ -193,69 +194,159 @@ public class LaDigitalAssister extends FlowInputs {
   
   private String personalSituationDisability;
 
+  // Income
+  private String householdSearchingForJob;
+
+  @NotEmpty(message="{error.missing-general}")
+  private List<String> jobSearch;
+
+  private String workDisqualificationInd;
+
+  private String selfEmploymentIncome;
+
+  @NotBlank(message="{error.missing-general}")
+  private String householdMemberJobAdd;
 
-  @Money
-  @NotBlank
+  @NotBlank(message="{error.missing-general}")
+  private String employerName;
+
+  private String selfEmployed;
+
+  private String jobPaidByHour;
+
+  @Money(message="{error.invalid-money}")
+  @NotBlank(message="{error.missing-dollar-amount}")
   private String hourlyWage;
 
-  @Range(min=1, max=100)
-  @NotBlank
+  @Range(message="{error.invalid-range}", min=1, max=100)
+  @NotBlank(message="{error.missing-general}")
   private String hoursPerWeek;
```

Comment:
@bseeger - I looked into this one and it's unrelated to my PR but it is a fun find!

That page is using the Number template which is looking for the input key in the the input; however hours worked is nested so it can never find that value and therefore always defaults to blank: https://github.com/codeforamerica/la-doc-uploader/blob/738c4cfde27e76322cf0a3bd6c18264bebee4873/src/main/resources/templates/fragments/inputs/number.html#L27C15-L27C15


The number fragment is being used in the LA doc uploader so I think it will require a bit digging into to understand the implications of the change. SO for now, creating a follow up ticket (https://app.asana.com/0/1204547723915585/1206144609816303/f)